package.json

```diff
@@ -30,7 +30,7 @@
     "ts-node": "8.3.0",
     "tslint": "6.1.0",
     "typescript": "3.8.3",
-    "progress-sitefinity-adminapp-sdk": "13.1.7424",
```

Comment:
7424? Shouldn't it be 7427 (or 7426 because we have missed the previous merge)

Reply:
seems like you have missed to merge develop to your branch

Reply:
The last merge we did is for 7424.